octopus_sensing/devices/skinos_streaming.py

```diff
@@ -0,0 +1,264 @@
+# This file is part of Octopus Sensing <https://octopus-sensing.nastaran-saffar.me/>
+# Copyright © Nastaran Saffaryazdi 2020
```

Comment:
You can add your name here next to Nastaran if you want. Because you wrote this code, you can be the copyright holder as well.